website/docs/Pipelines/SlideSeq_Pipeline/README.md

```diff
@@ -0,0 +1,234 @@
+---
+sidebar_position: 1
+---
+
+# SlideSeq Overview
+
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [SlideSeq v1.0.0](https://github.com/broadinstitute/warp/releases?q=slideseq&expanded=true) | October, 2022 | Elizabeth Kiernan & Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
+
+![SlideSeq_diagram](./slide-seq_diagram.png)
```

Comment:
I'm wondering if instead of bead metrics, we say either bead barcode or just barcode metrics? It seems like people at times just refer to barcodes to refer to the cell or bead barcode.

Reply:
If that's how it's most commonly referred to then I think that makes sense

Reply:
Updated